diesel/src/lib.rs

```diff
@@ -1,6 +1,17 @@
-//! Diesel is an ORM and query builder designed to reduce the boilerplate for database
-//! interactions. [A getting started guide](http://diesel.rs/guides/getting-started/) can be
-//! found on our website.
+//!# Diesel
+//!
+//! Diesel is an ORM and query builder designed to reduce the boilerplate for database interactions. 
+//!
+//! * [Homepage](http://diesel.rs/)
+//! * [Getting Started Guide](http://diesel.rs/guides/getting-started/)
+//! * [Source Code](https://github.com/diesel-rs/diesel/)
+//!
+//!## Crates
```

Comment:
Not sure this is the best heading, but this list needs a heading. Suggestions?

Reply:
[Rocket uses 'Libraries'](https://api.rocket.rs/rocket/) which I think is what I would suggest here. Super stoked for this to be done btw.

Reply:
"Libraries" sounds nicer, but is a CLI tool really a library?

Reply:
Good point. Hm. I'd almost suggest `Crates & Libraries` but there's only 3 things.

Reply:
So I spend a surprisingly long amount of time reading the wikipedia pages for gasoline and diesel to see what the things that make up the fuel are called… and all I found was "component"

Reply:
`Tools & Libraries`? 🤔 I think what you have written does a good job (Crates as the heading). Technically, each one of those "components" is a crate and there is a short description detailing its purpose. I don't see any ambiguous language that would confuse someone or inhibit them from getting to the correct documentation from your links. 

... although, they are "containers" that "hold diesel" code. We can call them `Tanks`.. 😆